### DISCUSSION TITLE
info: CrowdSec is an open source replacement for fail2ban.

### DISCUSSION THREAD

**Discussion by rjc3rd** [Ahwxorg/LibreY#162] [General]
Of course you should be running UFW on your server, but I also added this.

https://github.com/crowdsecurity/crowdsec

TLDR
CrowdSec is a free, modern & collaborative behavior detection engine, coupled with a global IP reputation network. It stacks on fail2ban's philosophy but is IPV6 compatible and 60x faster (Go vs Python), it uses Grok patterns to parse logs and YAML scenarios to identify behaviors. CrowdSec is engineered for modern Cloud / Containers / VM-based infrastructures (by decoupling detection and remediation). Once detected you can remedy threats with various bouncers (firewall block, nginx http 403, Captchas, etc.) while the aggressive IP can be sent to CrowdSec for curation before being shared among all users to further improve everyone's security.

**Comment by rjc3rd** [comment]
It's super easy to install.
The installation runs a simple script that enables it to run automatically.

**Comment by Ahwxorg** [comment]
Thank you, might be useful for someone!
